modules/dynamic-operator/controllers/kuberlogicservice-env/env.go

```diff
@@ -117,6 +121,53 @@ func (e *EnvironmentManager) SetupEnv(ctx context.Context) error {
 		}
 	}
 
+	registrySecret := &v1.Secret{
```

Comment:
We need to cover this in controller_test

Reply:
Done